geomstats/geometry/_my_manifold.py

```diff
@@ -0,0 +1,88 @@
+"""Template file to illustrate how to create a manifold in geomstats.
+
+For additional guidelines on how to contribute to geomstats, visit:
+https://geomstats.github.io/contributing.html#contributing-code-workflow
+
+Note: A manifold needs to be created with an associated test file.
+The test file for this manifold can be found at:
+tests/test__my_manifold.py.
+"""
+
+# Import the class(es) that MyManifold inherits from
+from geomstats.geometry.manifold import Manifold
+
+
+# This class inherits from the class Manifold.
+# Inheritance in geomstats means that the class MyManifold will reuse code
+# that is in the Manifold class.
+class MyManifold(Manifold):
+    r"""Give a one-liner description/definition of MyManifold.
+
+    For example: Class for Euclidean spaces.
+
+    Give a more detailed description/definition of MyManifold.
+    For example: By definition, a Euclidean space is a vector space of a given
+    dimension, equipped with a Euclidean metric.
+
+    List the parameters of MyManifold, i.e. the parameters given as inputs
+    of the constructor __init__.
+
+    For example:
+    Parameters
+    ----------
+    dim : int
+        Dimension of the manifold.
+    """
+
+    def __init__(self, dim, another_parameter, **kwargs):
+        super(MyManifold, self).__init__(dim)
+        self.another_parameter = another_parameter
+
+    # Implement the main methods of MyManifold, for example belongs:
+    def belongs(self, point):
```

Comment:
Mention that the signature of the method should match - when possible- the signature of the overriden method in the parent class. Here `belongs(self, point, atol=ATOL)`.

Reply:
Good point, done!